app/controllers/application_controller.rb

```diff
@@ -10,6 +10,6 @@ class ApplicationController < ActionController::API
   private
 
   def current_user
-    User.find(@decoded[:user_id]) if @decoded.present?
+    User.find_by(id: @decoded[:user_id]) if @decoded.present?
```

Comment:
find always used to field
```ruby
:id 
``` 
, why you change logic to
```ruby
find_by 
``` 
 for a specific field? 
 https://medium.com/@mizukilee/useful-active-record-methods-where-find-find-by-and-order-6890e598553e
 https://medium.com/@abhishektanwar9596/find-vs-find-by-in-rails-d319505a7b3

Reply:
I had a problem, when I deleted user, current user method tried to find user which I deleted and I got a problem like 404 error. When I used find_by I received nil, and it works in my case.